src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -79,34 +87,38 @@ function DomainSlider(props: Props): ReactElement {
       />
 
       <ToggleBtn
-        small
-        label={getAutoscaleLabel(isAutoMin, isAutoMax)}
-        icon={FiZap}
-        value={isAutoscaling}
+        iconOnly
+        label="Edit domain"
+        icon={MdTune}
```

Comment:
I feel the icon is not really appropriate here as it shows sliders.

Perhaps `MdEdit` or `FiEdit` would be more manifest ?

Reply:
Yeah, but those are really ugly on their own... Do you feel strongly about it? The icon is not meant to describe the UI but the fact that the domain can be "tuned". As long as users understand that... 🤷

Reply:
> Yeah, but those are really ugly on their own...

Really ? I like `FiEdit`
![image](https://user-images.githubusercontent.com/42204205/110782356-5426bf80-8267-11eb-9366-9841d2c2fdea.png)



> The icon is not meant to describe the UI but the fact that the domain can be "tuned".

I get you but the icon aspect is really at odds with the fact that clicking on it toggles editing. Well in any case:

> Do you feel strongly about it? 

No :smile:

Reply:
`FiEdit` isn't too bad, but it looks wrong visually because the square outline is not centred in the button (and I think tweaking the alignment would require reducing the icon's size too much).

But I'm not giving up yet! Here are a couple of alternatives (note that I can adjust the color/stroke to make some of these less imposing):

![image](https://user-images.githubusercontent.com/2936402/110784282-ce584380-8269-11eb-8b67-6b5f360e6ed5.png)

![image](https://user-images.githubusercontent.com/2936402/110784773-65250000-826a-11eb-818f-4e2e49adde0d.png)

![image](https://user-images.githubusercontent.com/2936402/110784876-7ec64780-826a-11eb-8d23-12ae2d434a29.png)

![image](https://user-images.githubusercontent.com/2936402/110785112-d8c70d00-826a-11eb-8dfc-c90116ca8a79.png)

![image](https://user-images.githubusercontent.com/2936402/110785163-e7adbf80-826a-11eb-8250-af00ff64823b.png)

![image](https://user-images.githubusercontent.com/2936402/110785295-1330aa00-826b-11eb-89a9-f3b36482a768.png)

... and my new favourite: 

![image](https://user-images.githubusercontent.com/2936402/110785346-26437a00-826b-11eb-9fe2-b5179ca5e4cc.png)

Reply:
With the `small` variant applied to the toggle:

![image](https://user-images.githubusercontent.com/2936402/110786533-8ab30900-826c-11eb-928b-e13a72325e7b.png)